occrp/templates/story.html

```diff
@@ -125,3 +125,52 @@
 </div>
 
 {% endblock %}
+{% block extra_javascript %}
+<script type="text/javascript">
+    function make_source(url) {
+        return function (request, response) {
+            jQuery.get(url, {
+                q: request.term
+            }, function (data) {
+              // Return an array of strings as a response.
+              var suggestions = []
+              if (data) {
+                $.each(data, function(index, value) {
+                  suggestions.push(value.suggestion)
+                });
+              }
+
+              response(suggestions);
+            });
+        }
+    };
+
+    function autocomplete_select() {
+        return function( event, ui ) {
+          console.log("Selected: " + ui.item.value);
```

Comment:
Did you mean to do something other than logging out the value here?

Reply:
No, I wanted that for testing...but it's useless. I'll clean it out.